README.md

```
@@ -1,92 +1,125 @@
-[Drizzle ORM](https://orm.drizzle.team)
+   ```bash
+   npx drizzle-kit generate
+   npx drizzle-kit migrate
+   ```
 
-[PostgreSQL](https://postgresql.org)
+   This repo extends the existing accounts table but doesn't create it initially.
+  
+   When forking:
+   1. Create an accounts table with the schema defined in `database/accounts-schema.ts`
+   2. Run migrations to add related authentication tables 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)